src/components/common/Button/Button.js

```diff
@@ -18,6 +26,14 @@ const Button = ({ children, variant, noHover, className: propClassName, ...props
     Comp = 'div';
   }
 
+  if (favorite) {
+    classes.push(styles.favorite);
+  }
+
+  if (compare) {
+    classes.push(styles.compare);
+  }
+
```

Comment:
Please use `clsx` package for conditional applying of classes – https://www.npmjs.com/package/clsx. It will be much cleaner that way.